misc/python/materialize/checks/owners.py

```diff
@@ -89,7 +91,12 @@ def _can_run(self) -> bool:
 
     def initialize(self) -> Testdrive:
         return Testdrive(
-            "> CREATE ROLE owner_role_01 CREATEDB CREATECLUSTER"
+            dedent(
+                """
+                > CREATE ROLE owner_role_01
+                > GRANT CREATEDB, CREATECLUSTER ON SYSTEM TO owner_role_01
```

Comment:
@def- I'm pretty sure I need to format this as something like:
```
if version<5900:
  ALTER ROLE owner_role_01 CREATEDB CREATECLUSTER
else:
  GRANT CREATEDB, CREATECLUSTER ON SYSTEM TO owner_role_01
```

How would I write that?

Reply:
Or we can update `_can_run` to require `v0.59.0-dev`.

Reply:
I'd prefer not to drop support for older versions, otherwise we are also not testing upgrades from < 0.59.

Like this (untested):
```
>[version<5900] ALTER ROLE owner_role_01 CREATEDB CREATECLUSTER
>[version>=5900] GRANT CREATEDB, CREATECLUSTER ON SYSTEM TO owner_role_01
```

Reply:
Done, looks like it's working.